executor/show.go

```diff
@@ -583,153 +583,166 @@ func (e *ShowExec) fetchShowCreateTable() error {
 
 	// TODO: let the result more like MySQL.
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "CREATE TABLE %s (\n", escape(tb.Meta().Name, sqlMode))
-	var pkCol *table.Column
-	var hasAutoIncID bool
-	for i, col := range tb.Cols() {
-		fmt.Fprintf(&buf, "  %s %s", escape(col.Name, sqlMode), col.GetTypeDesc())
-		if col.IsGenerated() {
-			// It's a generated column.
-			fmt.Fprintf(&buf, " GENERATED ALWAYS AS (%s)", col.GeneratedExprString)
-			if col.GeneratedStored {
-				buf.WriteString(" STORED")
-			} else {
-				buf.WriteString(" VIRTUAL")
-			}
-		}
-		if mysql.HasAutoIncrementFlag(col.Flag) {
-			hasAutoIncID = true
-			buf.WriteString(" NOT NULL AUTO_INCREMENT")
-		} else {
-			if mysql.HasNotNullFlag(col.Flag) {
-				buf.WriteString(" NOT NULL")
+	if !tb.Meta().IsView() {
```

Comment:
Can we add another function like "fetchShowCreateTable4View()", then:
```go
if tb.Meta().IsView() {
	fetchShowCreateTable4View(&buf)
	e.appendRow([]interface{}{tb.Meta().Name.O, buf.String()})
	return nil
}
```

Thus the original operation to prepare the result for normal table is unchanged, the code diff can be minimized.